pandas/io/parsers/readers.py

```diff
@@ -434,15 +436,19 @@
 )
 
 
-_c_parser_defaults = {
-    "delim_whitespace": False,
-    "na_filter": True,
-    "low_memory": True,
-    "memory_map": False,
-    "float_precision": None,
-}
+class _c_parser_defaults(Enum):
```

Comment:
For a TypedDict we would have needed a declaration and then the actual dict - using an Enum is much simpler here.

Reply:
Switch to the more verbose TypedDict: duplicate values in the enum